test/util.jl

```diff
@@ -1,15 +1,15 @@
 @testset "Forward Restoration" begin
     mn_data = build_mn_data(case5_restoration_strg, replicates=3)
     rop_result = PowerModelsRestoration.run_rop(mn_data, PowerModels.DCPPowerModel, cbc_solver)
-    PowerModelsRestoration.clean_solution(rop_result)
+    PowerModelsRestoration.clean_solution!(rop_result)
 
     @testset "ac forward case" begin
 
         PowerModels.update_data!(mn_data, rop_result["solution"])
         result = PowerModelsRestoration.run_forward_restoration(mn_data, PowerModels.ACPPowerModel, juniper_solver)
 
         @test result["termination_status"] == LOCALLY_SOLVED
-        @test isapprox(result["objective"], 6171.368; atol = 1e-2)
+        @test isapprox(result["objective"], 8421.368426450834; atol = 1e-2)
```

Comment:
Any idea why this objective is getting so much better?  More items are restored?

Reply:
The new objective function multiplies time_elapsed*repairs*old_objective.

The solutions are almost identical in terms of items repaired, when they are repaired, etc.

Reply:
Right, so if 5 items were repaired then we expect that the objective would increase by 5x?

Reply:
yes...that seems to be the case for the other objectives (just under 3x) but I didn't notice that this one is not as significant of a change.

Maybe a local minima because this is AC?  I'll look into it

Reply:
Actually, this case is running forward_restoration, the objective is a little less intuitive.  

Because it essentially runs MLD for X repair periods, the objective is reported for each sub-network.  Currently (and incorrectly for the long-term) the top-level report is the first network in the list, whichever that might be.